### PR TITLE
Add tests for service.go

### PR DESCRIPTION
**Pull request by Venkat19967**
**Description of the change:**
Added tests to Service.go

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Review by jmrodri — APPROVED** [review]
2 minor changes. Fix then we can merge.

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

File: internal/olm/operator/registry/configmap/service_test.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 The Operator-SDK Authors
```
Comment (camilamacedo86) [inline internal/olm/operator/registry/configmap/service_test.go:1]:
```suggestion
// Copyright 2020 The Operator-SDK Authors
```
nit

**Pull request merged**